.github/ISSUE_TEMPLATE/3.pro-support.yml

```
@@ -39,6 +39,25 @@ body:
   - type: textarea
     attributes:
       label: The problem in depth 🔍
+  - type: textarea
+    attributes:
+      label: Steps to reproduce 🕹
+      description: |
+        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
+        Include code to reproduce, if relevant (which it most likely is).
+
+        You can use the official codesandbox template of the [DataGrid](https://codesandbox.io/s/datagrid-v5-quick-start-54iz1) or [DataGridPro](https://codesandbox.io/s/datagridpro-v5-quick-start-5pjhc) as a starting point.
```

```suggestion
        You can use the official [DataGrid](https://codesandbox.io/s/datagrid-v5-quick-start-54iz1) or [DataGridPro](https://codesandbox.io/s/datagridpro-v5-quick-start-5pjhc) CodeSandbox template as a starting point.
```
Since this issue template is for Pro, do we need both links?

---

I have removed this, I wasn't thinking straight when I added it.